SafetyFirst/FiniteSeq.fs

```diff
@@ -136,6 +136,13 @@ module FiniteSeq =
   /// </summary>
   let inline concat (FSeq xs : FiniteSeq<FiniteSeq<'a>>) = fseq (xs |> LazyList.map (|FSeq|) |> LazyList.concat)
 
+  /// <summary>
+  /// Applies the given function to each element of the sequence and concatenates all the results.
+  /// Returned sequence is lazy, effects are delayed until it is enumerated.
+  /// </summary>
+  let inline collect (f : 'a -> FiniteSeq<'b>) (FSeq xs : FiniteSeq<'a>) = 
+    fseq (xs |> LazyList.map f |> LazyList.map (|FSeq|) |> LazyList.concat)
```

Comment:
for consistency, I feel like either this should reflect the above:
```
let g = f >> (|FSeq|)
fseq (xs |> LazyList.map g |> LazyList.concat)
```
or the above should inline it like we do here

Reply:
✔️